src/Compat.jl

```diff
@@ -803,6 +803,224 @@ if VERSION < v"1.11.0-DEV.1579"
     Iterators.cycle(xs, n::Integer) = Iterators.flatten(Iterators.repeated(xs, n))
 end
 
+# https://github.com/JuliaLang/julia/pull/39071
+if VERSION < v"1.12.0-DEV.2"  # might become wrong! PR will be backported to 1.11
```

Comment:
Seems safer in this case to use `isdefined`:
```suggestion
if !isdefined(Base, :logrange)  # VERSION < v"1.12.0-DEV.2" or appropriate 1.11.x after backporting
```

Reply:
Oh right, that's better.